package.json

```diff
@@ -13,7 +13,7 @@
     "adm-zip": "0.4.11",
```

Comment:
Can you add third party notice for both the tasks.

Reply:
Sure, will do that.